### PR TITLE
implementing best practice, optimising code

### PR DESCRIPTION
**Pull request by jakjakob**
removed "using namespaced std" [source](https://www.learncpp.com/cpp-tutorial/naming-collisions-and-an-introduction-to-namespaces/)

use \n instead of std::endl [source](https://www.learncpp.com/cpp-tutorial/introduction-to-iostream-cout-cin-and-endl/)

removed obvious chatGPT traces

optimisations explained in future comment (WIP)

### CODE REVIEW

**Comment by jakjakob** [comment]
use std::string_view instead of std::string somewhere

explicitly convert……… ?? why is the int changed into size_t? changing that

**Comment by jakjakob** [comment]
ah you are working on it that's why it is making so many conflicts

File: cake-bake.cpp [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
-
-// Python-like string multiplication
-string operator*(const string& s, size_t n) {
-    string result;
-    result.reserve(s.size()*n);
-    for(size_t i = 0; i < n; ++i) {
+//python-like string multiplication
+std::string operator*(std::string_view s, int n) {
+    std::string result;
+    result.reserve(s.length()*n);
```
Comment (jakjakob) [inline cake-bake.cpp:8]: chatGPT told me that I should use .length instead and so I did

**Review by captainexpo — APPROVED** [review]
lgtm!

**Pull request merged**